src/plugins/telemetry/server/telemetry_collection/get_local_license.ts

```diff
@@ -50,17 +50,17 @@ async function fetchLicense(opensearchClient: OpenSearchClient, local: boolean)
  *
  * In OSS we'll get a 400 response using the new opensearch  client.
  */
-async function getLicenseFromLocalOrMaster(opensearchClient: OpenSearchClient) {
-  // Fetching the local license is cheaper than getting it from the master node and good enough
+async function getLicenseFromLocalOrClusterManager(opensearchClient: OpenSearchClient) {
+  // Fetching the local license is cheaper than getting it from the cluster_manager node and good enough
```

Comment:
NIT: I think the comments would be `main` or `manager` node? But if not, it should definitely be `cluster manager` node.

Also, can you do me a favor and mark this as `deprecated` and create an issue to verify if this is in use? We shouldn't have any licensing checks. It's all telemetry.

Reply:
> NIT: I think the comments would be `main` or `manager` node? But if not, it should definitely be `cluster manager` node.
> Sure! Updated.
> Also, can you do me a favor and mark this as `deprecated` and create an issue to verify if this is in use? We shouldn't have any licensing checks. It's all telemetry.
Created: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1787